uk.co.compucorp.civicrm.hrcomments/tests/phpunit/CRM/HRComments/BAO/CommentTest.php

```diff
@@ -216,4 +213,45 @@ public function testIsDeletedCannotBeChangedWhenUpdatingExistingComment() {
     $this->assertEquals('SickRequest', $comment->entity_name);
     $this->assertEquals('This is a random sample comment', $comment->text);
   }
+
+  public function testValidateCommentCreatedAtDate() {
```

Comment:
I think that maybe we should test this using the `create` method instead of `validateParams`. The reason is to make sure that `validateParams` will be called inside the create method.

By changing that, please also change the test name to something more descriptive like `testCommnetCanNotBeCreatedWithACreatedDateLessThanTheCreatedDateOfTheLastCommentForThisEntity` (please feel free to use a better a name if you can think of one)